benchmark/fluid/README.md

```diff
@@ -29,9 +29,11 @@ Currently supported `--model` argument include:
     You can choose to use GPU/CPU training. With GPU training, you can specify
     `--gpus <gpu_num>` to run multi GPU training.
 * Run distributed training with parameter servers:
+    * see run_fluid_benchmark.sh as an example.
```

Comment:
Need a link here.

Reply:
Not sure this is needed. link can be broken and the file is just in this folder